packages/rs-drive-abci/src/abci/commit.rs

```diff
@@ -94,16 +96,20 @@ impl Commit {
 
         //todo: maybe cache this to lower the chance of a hashing based attack (forcing the
         // same calculation each time)
+        let quorum_hash = &self.inner.quorum_hash[..]
+            .try_into()
+            .expect("invalid quorum hash length");
```

Comment:
don't use expect. As a rule of thumb only use expect when there can be no doubt that something is the way it should be. No doubt meaning like the line above it. Also you can use it in tests.